services/QuillLMS/app/mailers/user_mailer.rb

```diff
@@ -146,6 +148,18 @@ def ell_starter_diagnostic_info_email(name, email)
     mail from: "The Quill Team <hello@quill.org>", to: email, subject: "ELL Starter Diagnostic Next Steps"
   end
 
+  def feedback_history_session_csv_download(email, data)
+    csv = CSV.generate(headers: true) do |csv_body|
+      csv_body << FEEDBACK_HISTORY_CSV_HEADERS
+      data.each do |row|
+        csv_body << [row["datetime"], row["session_uid"], row["conjunction"], row["attempt"], row["optimal"], (row['optimal'] || row['attempt'] == DEFAULT_MAX_ATTEMPTS).to_s, row["response"], row["feedback"], "#{row['feedback_type']}: #{row['name']}"]
```

Comment:
For readability, I like to break method calls with 3+ parameters into multiple lines:

```
  csv_body << [
    row["datetime"], 
    row["session_uid"], 
    row["conjunction"], 
    row["attempt"], 
    row["optimal"], 
    row['optimal'] || row['attempt'] == DEFAULT_MAX_ATTEMPTS, 
    row["response"], 
    row["feedback"], 
    "#{row['feedback_type']}: #{row['name']}"
  ]
```

Also, I think the `to_s` on the boolean is unnecessary since the `CSV#generate` should call to_s internally.